tests/test_bsdf.py

```diff
@@ -14,6 +14,11 @@
 from imageio.plugins import _bsdf as bsdf
 
 
+import pytest
+import sys
+xfail_big_endian = pytest.mark.xfail(sys.byteorder == 'big',
+                                     reason='expected failure on big-endian')
+
```

Comment:
Can you run `black` formatting on this file? That should make Travis happy :)

Reply:
done!